### DISCUSSION TITLE
PETG Tower temp labels

### DISCUSSION THREAD

**Discussion by msteele999** [5axes/Calibration-Shapes#39] [Q&A]
I want to test my PETG starting at 250 and working my way down by 5 degrees.

The object starts at 260C - am I able to modify the model and make it available thru the plugin?

Thanks for the great work on this!

Mark

**Comment by 5axes** [comment]
Yes .. You need to use OpenScad, install the software on your PC , modify the geomtry as you want and save with the same the model under a new stl file, using the same name

**Comment by msteele999** [comment]
where is the original model (that the plugin is pulling in) stored?

I have OpenScad already loaded for a couple of other projects I modify.

Thanks!

**Comment by 5axes** [comment]
at the same place as the stl file, in the **models** directory

**Reply by msteele999** [reply]
Thank you - I don't mean to be thick, but where is the models directory located? Is it under the Program Files directory for Cura, or somewhere else?

**Reply by 5axes** [reply]
In Cura use the function -> Help -> Show Configuration Folder 

![image](https://user-images.githubusercontent.com/11015345/119143098-831b9780-ba47-11eb-91f4-961c2e0919ea.png)

Then  **plugins\CalibrationShapes\models**

**Reply by msteele999** [reply]
Dude - thanks - and sorry!

You'd think I'd know that by now.
